src/renderers/dom/client/syntheticEvents/SyntheticMouseEvent.js

```diff
@@ -63,6 +63,32 @@ var MouseEventInterface = {
       event.pageY :
       event.clientY + ViewportMetrics.currentScrollTop;
   },
+  offsetX: function(event) {
+    return 'offsetX' in event ?
+    event.offsetX :
+    'srcElement' in event ?
+      ((e) => {
+        const target = e.srcElement;
+        const style = target.currentStyle || window.getComputedStyle(target, null);
+        const borderLeftWidth = parseInt(style.borderLeftWidth, 10);
+        const rect = target.getBoundingClientRect();
+        return parseInt((e.clientX - borderLeftWidth - rect.left), 10);
+      })(event) :
+      'undefined';
```

Comment:
I wasn't sure whether to use null or undefined. Was looking for some advice there.

I'm not sure what you mean by "on creation" & "on demand". It does execute on every MouseMove event. Should I remove the polyfill?